tests/event.py

```diff
@@ -457,3 +457,31 @@ def test_issue_92(self):
 
         assert e.begin == arrow.get('2016-10-04')
         assert e.end == arrow.get('2016-10-05')
+
+    def test_classification_input(self):
+        c = Calendar(cal33)
+        e = next(iter(c.events))
+        self.assertEqual('PUBLIC', e.classification)
+
+        c = Calendar(cal34)
+        e = next(iter(c.events))
+        self.assertEqual('PRIVATE', e.classification)
+
+    def test_classification_input_wrong_value(self):
+        with pytest.raises(ValueError):
+            Calendar(cal35)
+
+    def test_classification_output(self):
+        e = Event(name="Name", classification='PUBLIC')
+        self.assertIn("CLASS:PUBLIC", str(e).splitlines())
+
+        e = Event(name="Name", classification='PRIVATE')
+        self.assertIn("CLASS:PRIVATE", str(e).splitlines())
+
+    def test_classification_bool(self):
+        with pytest.raises(ValueError):
+            Event(name="Name", classification=True)
+
+    def test_classification_wrong_value(self):
+        with pytest.raises(ValueError):
+            Event(name="Name", classification='wrong_value')
```

Comment:
1. Missing tests for classification not set (maybe anti-pattern or do we want `e.classification` to be `None`)?
  2. Missing tests for other valid values (`CONFIDENTIAL`, `iana-token`, `x-name`).

Reply:
1. All existing test doesn't have a value for classification, there fore I thought no extra test is needed and in the RFC it is written
> If not specified in a component that allows this property, the default value is PUBLIC

2. missing tests are added with commit ab8dc7ffd21221a5bb0b8591f216a44dec55bfa1

Reply:
Thanks for the added tests.
Should still decide whether `e.classification` should return `"PUBLIC"`, `None`, or raise an exception.
@C4ptainCrunch Care to chime in?

Reply:
I would say that by default, if the `._classification` attribute is `None`, the `.classification` property should return `PUBLIC` as it's the default in the RFC.
(Note, if the `._classification` attribute is `None`, it should not be added to the properties of the VEVENT when serializing [aka here](https://github.com/C4ptainCrunch/ics.py/pull/178/files#diff-0831b8f95d6c3e878c59cb1bb8cdbdd3R655))

Reply:
@C4ptainCrunch You mean when reading (for example from file) or creating an event the `._classification` attribute should always be set to `PUBLIC`?

Reply:
What i'm saying is that `._classification` (the private attribute) should be `PUBLIC` only if explicitly setted either by the user or in the source file.
When creating a new event or when reading an event that does not have the `CLASSIFICATION` property, `._classification` should be `None`

Reply:
more test added with 0b1724b6ad772654fc78ec11c3e0f95f6a9b1ac8 
do we need to change more to resolve that comment?